recipes/dftbplus-tools/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "dftbplus" %}
+{% set version = "20.1" %}
+
+package:
+  name: {{ name|lower }}-tools
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/4f5907c38860683723b2c2cc16860e832b359f71.tar.gz
+  sha256: f7c294bc18098a8a5229196b4e783d800f870a16e3f217cdda736d99a9097e57
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install tools/dptools -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
```

Comment:
This should just be "numpy".  There is no numpy in host to be compatible with and since you are not compiling against numpy, there shouldn't be